### ISSUE TITLE
Possible redundancy in computer card-purchasing loop

### ISSUE THREAD

**Issue by flipdazed** [flipdazed/SoftwareDevelopment#40]
**Bugged Version** 0_0_0
**Lines** `232; 236`

**Issue**
The `money` loop appears to supersede the `cb` inner-loop.
Address this issue first then look at #39 (`cb` inconsistency): merge likely

```
if money > 0:
            cb = True
            templist = []
            print "Starting Money %s and cb %s " % (money, cb)
            while cb:
```

**Fix**
1. Check if loop is indeed redundant
2. Remove `cb` loop

**Comment by flipdazed** [comment]
will be fixed by class implementation - reassigned

**Comment by flipdazed** [comment]
moved as not really class implementation - more of an enhancement

**Comment by flipdazed** [comment]
closed as duplicate #39
